include/auxiliary.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+#include <basics.h>
+#include <config.h>
+#include <seqan/bam_io.h>
+#include <seqan/seq_io.h>
+#include <iostream>
+#include <memory>
+#include <thread>
+
+struct Auxiliary{
```

Comment:
Ok let's call this `input_output_information`. The file can still be called `auxiliary.hpp` since we have also prep_file_handels in here.